lib/plugins/orionUrls.js

```diff
@@ -39,6 +39,8 @@ module.exports = [
   ['POST', /\/(v1|ngsi10)\/unsubscribecontext$/, 'subscribe'],
   ['POST', /\/(v1\/registry|ngsi9)\/registercontext$/, 'register'],
   ['POST', /\/(v1\/registry|ngsi9)\/discovercontextavailability$/, 'discover'],
+  ['POST', /\/(ngsi10|v1)\/contexttypes$/, 'read'],
```

Comment:
Taking into account (https://docs.google.com/spreadsheets/d/1f4m624nmO3jRjNalGE11lFLQixnfCENMV6dc54wUCCg/edit#gid=0):

> Note 3: "/ngsi10" is an alias prefix for "/v1", while "/ngsi9" is an alias prefix for "/v1/registry" (versions previous to 0.16.0 only allows /ngsiX prefixes), except for new operations added since 0.15.0, which use only the "/v1" prefix.

and that the operation was introduced in 0.15.0, this will be only "v1" (not "ngsi10").